src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -125,25 +158,32 @@ protected function setup_repository_args( Tribe__Context $context = null ) {
 	 */
 	public function add_view_filters() {
 
-		add_filter( 'tribe_template_html:components/messages', [ $this, 'filter_template_done' ] );
-		add_filter( 'tribe_template_html:events/v2/day', [ $this, 'add_view' ] );
-		add_filter( 'tribe_template_html:events/v2/list', [ $this, 'add_view' ] );
-		add_filter( 'tribe_template_html:events/v2/month', [ $this, 'add_view' ] );
+		add_filter( 'tribe_template_html:events/v2/components/messages', [ $this, 'filter_template_done' ] );
+		//add_filter( 'tribe_template_html:events/v2/day', [ $this, 'add_view' ] );
+
+
+		//todo loading on the ical link goes below the existing view, although the existing view is not whitelisted so no longer shows.
+		//todo this also places it in the existing view wrap and prevents display issues with the footer padding.
```

Comment:
:no_entry_sign: No space found before comment text; expected "// todo this also places it in the existing view wrap and prevents display issues with the footer padding." but found "//todo this also places it in the existing view wrap and prevents display issues with the footer padding."

<sub>hash: 9d328e38735dbb564eaf6550b80f63c1</sub>